lib/smooth/core/network/Wifi.cpp

```diff
@@ -37,8 +37,11 @@ using namespace smooth::core;
 
 namespace smooth::core::network
 {
+    ip4_addr_t Wifi::ip = { 0 };
+
     Wifi::Wifi()
     {
+        this->ip.addr = 0;
```

Comment:
This line is no longer needed since it is initialized just above.

Reply:
ups, I missed that.